ingestion/src/metadata/data_quality/source/test_suite.py

```diff
@@ -104,8 +106,16 @@ def test_connection(self) -> None:
 
     def _iter(self) -> Iterable[Either[TableAndTests]]:
         table: Table = self._get_table_entity()
-
         if table:
+            source_type = table.serviceType.value.lower()
+            if source_type.startswith("custom"):
+                logger.warning(
+                    "Data quality tests might not work as expected with custom sources"
+                )
```

Comment:
I thought this was not supported at all. Have you checked if we could actually run tests?

Reply:
no. but current behavior will not stop a user from running tests on any table.

Reply:
Make sense. It should go through indeed.